frontend/src/components/PsyListing/PsyListing.jsx

```diff
@@ -56,6 +56,7 @@ const PsyListing = () => {
       description={`Il y a actuellement ${psychologists.length} partenaires du dispositif d‘accompagnement.
       La liste est mise à jour quotidiennement, revenez la consulter si vous n‘avez pas pu trouver de psychologue.`}
       background="yellow"
+      dataTestId="psyListPage"
```

Comment:
data-test-id to be coherent ?

Reply:
```suggestion
      data-test-id="psyListPage"
```

Reply:
If I rename it `data-test-id` I won't be able to call the prop in Page component like this :confused: 
It's done the same way for `DateInput`.